cfbs/commands.py

```diff
@@ -1051,7 +1051,7 @@ def _compare_list(a, b):
         if (
             not isinstance(a, dict)
             or not isinstance(b, dict)
-            or not _compare_dict(a, b, ignore={"response"})
+            or not _compare_dict(a, b, ignore=set({"response"}))
```

Comment:
So this is creating a set from a dict. Would make more sense to use a tuple or a list here.

Reply:
@vpodzime no, curly braces in python can mean both set or dict. This change was not needed. When there is just one or more values, separated by commas, no colons to indicate key-value pairs, it's a set.

Reply:
```python
>>> type({"response"}).__name__
'set'
```